### DISCUSSION TITLE
Bump mysql minimum version to 8?

### DISCUSSION THREAD

**Discussion by pat-s** [woodpecker-ci/woodpecker#2392] [General]
Because of https://github.com/woodpecker-ci/woodpecker/pull/2232.

There will most likely never be a arm64 image for mysql 5.x and this DB type/version might anyhow not be the most recommended one anyhow?

**Comment by 6543** [comment]
@pat-s yes - please do so :)

**Comment by pat-s** [comment]
Done!
